lib/helpers/mail/Mail.php

```diff
@@ -948,12 +948,18 @@ class Mail implements \JsonSerializable
 
     public function __construct($from = null, $subject = null, $to = null, $content = null)
```

Comment:
I think we should remove the ability to set these to null since the helper will not work unless all of these parameters are set.

Reply:
What if the user wants to create the object and then add personalizations separately, e.g. for sending to many recipients?

Reply:
Also what about using transactional templates where the subject isn't set programmatically at all?

Reply:
@Tzafra,

Would you mind opening up a new issue so that we can get better visibility on this issue?